contracts/Whitelist.sol

```diff
@@ -38,6 +66,15 @@ contract Whitelist is Ownable {
         return whitelistedProduct[productHash];
     }
 
+    /**
+     * @notice check if an otoken is whitelisted
+     * @param _otoken otoken address
+     * @return boolean, true if otoken is whitelisted
+     */
+    function isWhitelistedOtoken(address _otoken) external view returns (bool) {
+        return whitelistedOtoken[_otoken];
```

Comment:
`isValidOtoken` make more sense to me. It's not like we only whitelist several otokens out of all the otokens, we literally whitelist every valid token created by the factory.

Reply:
yeah thought better to use same terminology everywhere, "Whitelist module, whitelist, isWhitelisted", even if we whitelist all token, it is still the same as valid as we are marking all token as valid. I think better to use same terminology.

Reply:
Let's use this for now, could easily change it back if we end up having different decision.

Reply:
+1 on Anton's comment